app/scripts/modules/core/src/search/infrastructure/RecentlyViewedItems.tsx

```diff
@@ -83,6 +85,10 @@ export class RecentlyViewedItems extends React.Component<IRecentlyViewedItemsPro
     this.updateRecentItems();
   }
 
+  private handleResultClick(categoryName: string): void {
+    ReactGA.event({ category: 'Global Search', action: `Recent item selected from ${categoryName}` });
```

Comment:
not sure if this makes sense to me — `RecentlyViewItems` renders both the Global Search recent results and the normal search results, and right now it looks like this handler is getting called for the normal search results with the `Global Search` category, whereas the actual global search component is using its own handler and ignoring this one altogether. Couple of things:

1) It feels like we probably want to differentiate (likely via category?) between normal search and global search, so we know where in the UI people are interacting with search results. That'd mean we'd end up with a `SearchResultPod` specific callback in addition to the `GlobalSearchRecentItems` specific callback we currently have, and we'd probably eliminate the common one here.
2) If we *don't* end up differentiating between them and keep a common event logger here, we should at least have the `onResultClick` for `GlobalSearchRecentItems` call through to this shared function after it calls `this.hideDropdown`.

Reply:
You're right, this was bad copy/paste. I will fix the category in a followup PR...